cassandra/src/main/scala/com/stratio/crossdata/connector/cassandra/statements/CreateTableStatement.scala

```diff
@@ -0,0 +1,158 @@
+import org.apache.spark.sql.types.{TimestampType => SparkSqlTimestampType}
+
+object CreateTableStatement {
+
+  val PRIMARY_SINGLE = 1
+  val PRIMARY_COMPOSED = 2
+  val PRIMARY_AND_CLUSTERING_SPECIFIED = 3
+
+}
+
+case class CreateTableStatement(tableName: String,
+                                schema: StructType,
+                                options: Map[String, String]
+                               ) {
+
+  override def toString(): String = {
+    val cqlCommand = StringBuilder.newBuilder
+    cqlCommand.append("CREATE TABLE ")
+
+    if (ifNotExists) {
+      cqlCommand.append("IF NOT EXISTS ")
+    }
+
+    cqlCommand.append(keyspace).append(".").append(tableName)
+
+    cqlCommand.append(" (")
+    schema.fields.foreach { field: StructField =>
+
+      val cassandraDataType = fromSparkSqlType(field.dataType)
+      cqlCommand.append(field.name).append(" ").append(cassandraDataType.cqlTypeName).append(", ")
+    }
+    cqlCommand.append(getPKString).append(")")
+
+
+    cqlCommand.toString()
+  }
+
+  /**
+    * The type of primary key. Accepted values are:
+    * <ul>
+    * <li>1: If the primary key contains a single column.</li>
+    * <li>2: If the primary key is composed of several columns but it does not contain a clustering
+    * key.</li>
+    * <li>3: If both the primary key and clustering key are specified.</li>
+    * </ul>
+    */
+  lazy val primaryKeyType: Int = {
+    options.getOrElse("primaryKeyType", "1").toInt
+  }
+
+  /**
+    * Obtain the composite primary key.
+    * @return a string with the primary key.
+    */
+  lazy val getPKString:String = {
+    import CreateTableStatement._
+
+    val primaryKeyString = StringBuilder.newBuilder
+    primaryKeyString.append("PRIMARY KEY (")
+
+    var colon = ""
+    primaryKeyType match {
+      case PRIMARY_COMPOSED | PRIMARY_SINGLE =>{
+        primaryKeys.foreach{ primkey =>
+          primaryKeyString.append(colon).append(primkey)
+          colon = ","
+        }
+        primaryKeyString.append(")")
+      }
+      case PRIMARY_AND_CLUSTERING_SPECIFIED => {
+        //TODO
+        ???
+      }
+    }
+
+
+    primaryKeyString.toString()
+  }
+
+  lazy val ifNotExists: Boolean = {
+    options.contains("ifNotExist")
+  }
+
+  lazy val keyspace: String = {
+    require(options.contains(CassandraDataSourcePrimaryKeyNameProperty), s"$CassandraDataSourcePrimaryKeyNameProperty required when use CREATE EXTERNAL TABLE command")
+    options.get("keyspace").get
+  }
+
+  lazy val primaryKeys:Seq[String] = {
+    require(options.contains(CassandraDataSourcePrimaryKeyNameProperty), s"$CassandraDataSourcePrimaryKeyNameProperty required when use CREATE EXTERNAL TABLE command")
+    options.get(CassandraDataSourcePrimaryKeyNameProperty).get.split(",")
+  }
+
+
+  /** Returns natural Cassandra type for representing data of the given Spark SQL type */
+  def fromSparkSqlType(dataType: SparkSqlDataType): ColumnType[_] = {
```

Comment:
This method could be extracted to a common class in order to be used for later usages